### PR TITLE
show small Join and CC icons for aggregation without breakout

### PR DESCRIPTION
**Pull request by uladzimirdev**
[Epic](https://github.com/metabase/metabase/issues/47911)

Add `join` and `custom column` small icons after summarize step if there is aggregation without a breakout

https://github.com/user-attachments/assets/71d1b6c0-60f2-43df-a20d-79aaa549d8f4

### CODE REVIEW

File: e2e/test/scenarios/custom-column/cc-shortcuts-combine.cy.spec.ts [hunk 1]
```diff
@@ -111,26 +111,6 @@ describe("scenarios > question > custom column > expression shortcuts > combine"
       cy.findByLabelText("Separator").should("have.value", "");
     });
   });
-
-  it("should be possible to edit a previous stages' columns when there is an aggregation (metabase#43226)", () => {
```
Comment (uladzimirdev) [inline e2e/test/scenarios/custom-column/cc-shortcuts-combine.cy.spec.ts:115]: looks to me 1:1 with 

https://github.com/metabase/metabase/blob/master/e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts#L344-L363

File: e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts [hunk 2]
```diff
@@ -348,7 +349,8 @@ describe("scenarios > question > custom column > expression shortcuts > combine"
 
     popover().findByText("Count of rows").click();
 
-    addCustomColumn();
+    // add custom column
+    cy.findAllByTestId("action-buttons").first().icon("add_data").click();
```
Comment (uladzimirdev) [inline e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts:353]: now we have 2 blocks of action buttons, so helper doesn't handle this case

File: e2e/test/scenarios/custom-column/custom-column-reproductions.cy.spec.js [hunk 3]
```diff
@@ -96,7 +96,7 @@ describe("issue 13289", () => {
     cy.button("Done").click();
   });
 
-  it("should allow 'zoom in' drill-through when grouped by custom column (metabase#13289) (metabase#13289)", () => {
```
Comment (uladzimirdev) [inline e2e/test/scenarios/custom-column/custom-column-reproductions.cy.spec.js:99]: issue name was duplicated

File: e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts (hunk 2 above)
Comment (romeovs) [reply e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts:353]: Can we create a helper for this case? I think we should create a habit of doing that whenever we introduce new UI components.

**Review by romeovs — APPROVED** [review]
LGTM but I think we should create a new e2e test helper for the new UI components that were introduced.

File: e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts (hunk 2 above)
Comment (uladzimirdev) [reply e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts:353]: IDK, its needed only in two places so far, also we have another helper imported here `addCustomColumn` that's why I just inlined those called to clearly show we're interested only in the first `action-buttons`

Comment (uladzimirdev) [reply e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts:353]: to clarify: we do not introduce new components here, we only have cases when two groups of action buttons are rendered at the same time

Comment (romeovs) [reply e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts:353]: Yep not a blocker for me, I agree that it's maybe not enough of a "new component" to warrant a separate helper.

**Pull request merged**